documentation/tutorial.md

```diff
@@ -1,28 +1,37 @@
-# Charmed MongoDB tutorial:
-Ready to try out Charmed MongoDB? This tutorial will guide you through the steps to get Charmed MongoDB up and running. 
+# Charmed MongoDB tutorial
+The Charmed MongoDB Operator delivers automated operations management from day 0 to day 2 on the [MongoDB Community Edition](https://github.com/mongodb/mongo) document database. It is an open source, end-to-end, production-ready data platform on top of juju. As a first step this tutorial show you how to get Charmed MongoDB up and running, but the tutorial does not stop there. Through this tutorial you will learn a variety of operations, everything from adding replicas to advanced operations such as enabling TLS. Specifically this tutorial will cover:
```

Comment:
```suggestion
The Charmed MongoDB Operator delivers automated operations management from day 0 to day 2 on the [MongoDB Community Edition](https://github.com/mongodb/mongo) document database. It is an open source, end-to-end, production-ready data platform on top of Juju. As a first step this tutorial shows you how to get Charmed MongoDB up and running, but the tutorial does not stop there. Through this tutorial you will learn a variety of operations, everything from adding replicas to advanced operations such as enabling TLS. Specifically this tutorial will cover:
```

Reply:
I do have a few questions about this paragraph. It's likely your users will have a higher baseline of knowledge than I do about the product, but I wasn't sure what the following meant:
- day 0 to day 2
- TLS (in a tutorial I'd always expand the first use of every acronym, in case there are others it could be confused for)
- adding replicas

- I'd also add a link to the Juju page where you mention Juju
- Is there a website for the MongoDB Community Edition? I got a bit confused to be taken to GitHub

Reply:
All of these are great suggestions! I've updated each of these accordingly Except for:
- "adding replicas", this is a fundamental concept of MongoDB that we will expect the user to know
- and the URL for MongoDB Community Edition, the offical site for MongoDB Community Edition is https://www.mongodb.com/try/download/community and I think the download page is not as informative as the github page.

Reply:
Do you think we should mention that we expect a familiarity with MongoDB?

Reply:
Personally, I'm always in favour of stating expectations up front. In the case of a tutorial that expects the reader to know something before they start, telling them what topics they need to be familiar with is helpful as it sets their expectations about whether or not they can currently complete the tutorial. If they can't yet, I'd say "be opinionated" and point them in the direction of the best resources to help them get to the right level. In my experience, if the reader has to go away and find out these things on their own after struggling with a tutorial they don't understand well (i.e. if the entry barrier is too high) they often don't come back - so I'm a proponent of making it easy for them :)